app/controllers/api/mixins/pictures.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module Mixins
+    module Pictures
+      def fetch_picture(resource)
+        format_picture_response(resource.picture)
+      end
+
+      def format_picture_response(picture)
```

Comment:
I need to get that serializers PR finished ;)

Reply:
@imtayadeway yes please! 😛